schemas/Account.json

```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Admin users may temporarily disable an account by setting this flag",
```

Comment:
What if we call the field `is_disabled` for consistency with the `is_admin` field?
I'm inclined to use the past tense for this field for consistency across put/get/db column names (when a user GETs an account or looks in the db, they want to see the `is_disabled` flag, not the `disable` flag).